File: helfomat-import/src/main/java/de/helfenkannjeder/helfomat/ImportingApplication.java
package de.helfenkannjeder.helfomat;

import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.WebApplicationType;
import org.springframework.boot.autoconfigure.SpringBootApplication;

/**
 * @author Valentin Zickner
 */
@SpringBootApplication
@EnableBatchProcessing
public class ImportingApplication {

    public static void main(String[] args) {
        SpringApplication springApplication = new SpringApplication(ImportingApplication.class);
        springApplication.setWebApplicationType(WebApplicationType.NONE);
        springApplication.run(args);
    }
}

File: helfomat-import/src/main/kotlin/de/helfenkannjeder/helfomat/ImportingApplication.kt
package de.helfenkannjeder.helfomat

import org.springframework.batch.core.configuration.annotation.EnableBatchProcessing
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.retry.annotation.EnableRetry
import kotlin.system.exitProcess

/**
 * @author Valentin Zickner
 */
@SpringBootApplication
@EnableBatchProcessing
@EnableRetry
open class ImportingApplication

fun main(args: Array<String>) {
    val context = runApplication<ImportingApplication>(*args)
    exitProcess(SpringApplication.exit(context))
}
